lib/mailer_previews/user_mailer_preview.rb

```diff
@@ -36,7 +36,7 @@ def work_collaborator
 
   def nightly_user_activity
     user = User.find_by(login: 'admin')
-    UserMailer.nightly_user_activity(user)
+    UserMailer.nightly_user_activity(UserMailer::Activity.build(user))
```

Comment:
Using dependency injection here lets us be selective about the types of activities we want to use with the nightly mailer. It also makes testing _much_ easier.